CONTRIBUTING.md

```diff
@@ -1,12 +1,78 @@
-## Before you submit a new issue
-
-* Check if there's a simple solution in the
-  [Troubleshooting](https://github.com/npm/npm/wiki/Troubleshooting)
-  wiki.
-* [Search for similar
-  issues](https://github.com/npm/npm/search?q=Similar%20issues&type=Issues).
-* Ensure your new issue conforms to the [Contributing
-  Guidelines](https://github.com/npm/npm/wiki/Contributing-Guidelines).
-
-Participation in this open source project is subject to the [npm Code
-of Conduct](http://www.npmjs.com/policies/conduct).
+# npm CLI Contributor Guide
+
+## Introduction
+
+Welcome to the npm CLI Contributor Guide! This document outlines the npm CLI repository's process for community interaction and contribution. This includes the issue tracker, pull requests, wiki pages, and, to a certain extent, outside communication in the context of the npm CLI. It defines roles, responsibilities, and procedures, and is an entry point for anyone wishing to contribute their time and effort to making npm a better tool for the JavaScript community!
+
+All interactions in the npm repository are covered by the [npm Code of Conduct](https://www.npmjs.com/policies/conduct)
+
+## Roles
+
+There are three main roles for people participating in the npm issue tracker. Each has a specific set of abilities and responsibilities: [Community members](#community-members), [Collaborators](#collaborators), and [npm, Inc employees](#npm-inc-employees). 
+
+Failure to comply with the expected responsibilities of each role, or violating the Code of Conduct will result in punitive action relative to the transgression, ranging from a warning to full removal from the project, at the discretion of npm employees. 
+
+### Community Members
+
+This includes anyone who may show up to the npm/npm repo with issues, PRs, comments etc. They may not have any other involvement with npm. 
+
+#### Abilities
+
+* Open issues and PRs
+* Comment on issues and PRs
+
+#### Responsibilities
+
+* Comment on issues when they have a reference to the answer. 
+* If community members aren't sure they are correct and don't have a reference to the answer, it's best to leave the issue and try another one.  
```

Comment:
I wonder if we can be a bit stronger here

"It's best to leave the issue"

Something like "please leave the issue and try another one". I'd like to get across that you _should not try_ if you don't know the answer.